### PR TITLE
Module landing page: scrollbar styles

### LINKED ISSUE #154: web/css/css_scrollbars/ -
https://github.com/mdn/content/pull/26400

### PR DESCRIPTION
**Pull request by estelle**
part of https://github.com/openwebdocs/project/issues/147

fixes https://github.com/openwebdocs/project/issues/154

### CODE REVIEW

**Comment by dipikabh** [comment]
I'll take a look at this tomorrow 👍

File: files/en-us/web/css/css_scrollbars/index.md [hunk 1]
```diff
@@ -1,67 +1,101 @@
...
 
-## Browser compatibility
-
-{{Compat}}
-
 ## See also
 
-- {{CSSxRef("::-webkit-scrollbar")}}
+- [CSS overflow module](/en-US/docs/web/CSS/CSS_overflow)
+- [CSS scroll snap module](/en-US/docs/web/CSS/CSS_Scroll_Snap)
```
Comment (teoli2003) [inline files/en-us/web/css/css_scrollbars/index.md:99]: Capitalization matters (see flaws)
```suggestion
- [CSS Scroll Snap module](/en-US/docs/Web/CSS/CSS_Scroll_Snap)
```

File: files/en-us/web/css/css_scrollbars/index.md [hunk 2]
```diff
@@ -1,67 +1,101 @@
...
 {{Specifications}}
 
-## Browser compatibility
-
-{{Compat}}
-
 ## See also
 
-- {{CSSxRef("::-webkit-scrollbar")}}
+- [CSS overflow module](/en-US/docs/web/CSS/CSS_overflow)
```
Comment (teoli2003) [inline files/en-us/web/css/css_scrollbars/index.md:98]:
```suggestion
- [CSS Overflow module](/en-US/docs/Web/CSS/CSS_Overflow)
```

File: files/en-us/web/css/css_scrollbars/index.md [hunk 3]
```diff
@@ -1,67 +1,101 @@
...
+- [CSS scroll snap module](/en-US/docs/web/CSS/CSS_Scroll_Snap)
+- [`@media (forced-colors: active)`](/en-US/docs/Web/CSS/@media/forced-colors) sets {{cssxref("scrollbar-color")}} to 'auto'
+- {{Experimental_Inline}} The CSS scroll timeline {{cssxref('scroll-timeline-name')}} and {{cssxref('scroll-timeline-axis')}} properties, along with the {{cssxref('scroll-timeline')}} shorthand, to create animations tied to the scroll offset of a {{glossary("scroll container")}}.
```
Comment (teoli2003) [inline files/en-us/web/css/css_scrollbars/index.md:101]:
```suggestion
- {{{cssxref('scroll-timeline-name')}}, {{cssxref('scroll-timeline-axis')}}, and {{cssxref('scroll-timeline')}} 
```

**Review by teoli2003 — DISMISSED** [review]

**Review by dipikabh — CHANGES_REQUESTED** [review]
Added my comments

File: files/en-us/web/css/css_scrollbars/index.md [hunk 4]
```diff
@@ -1,67 +1,99 @@
...
-  - css.properties.scrollbar-width
-  - css.properties.scrollbar-color
+spec-urls: https://w3c.github.io/csswg-drafts/css-scrollbars/
 ---
 
-{{CSSRef}}
+{{CSSRef}}{{SeeCompatTable}}
 
-**CSS Scrollbars** standardizes the obsolete scrollbar color properties introduced in 2000 by Windows IE 5.5.
+The **CSS scrollbars styling** module defines properties that provide visual styling of scrollbars, including controls for their color and width.
```
Comment (estelle) [inline files/en-us/web/css/css_scrollbars/index.md:10]:
```suggestion
The **CSS scrollbars styling** module defines properties that provide visual styling of scrollbars, including controls for the color and width of scrollbar tracks, the background of the scrollbar, and the color of scrollbar thumbs, the draggable handle of the scrollbar.
```

File: files/en-us/web/css/css_scrollbars/index.md [hunk 5]
```diff
@@ -1,58 +1,93 @@
...
 browser-compat:
-  - css.properties.scrollbar-width
   - css.properties.scrollbar-color
 ---
 
-{{CSSRef}}
+{{CSSRef}}{{SeeCompatTable}}
 
-**CSS Scrollbars** standardizes the obsolete scrollbar color properties introduced in 2000 by Windows IE 5.5.
+The **CSS scrollbars styling** module defines properties that provide visual styling of scrollbars, including controls for the color and width of scrollbar tracks, the background of the scrollbar, and the color of scrollbar thumbs, the draggable handle of the scrollbar.
```
Comment (dipikabh) [inline files/en-us/web/css/css_scrollbars/index.md:12]:
```suggestion
The **CSS scrollbars styling** module defines properties that you can use for visual styling of scrollbars. You can  customize the width of the scrollbar as required. You can also customize the color of the scrollbar _track_, which is the background of the scrollbar, and the color of the scrollbar _thumb_, which is the draggable handle of the scrollbar.
```
I guess the width definition is for the scrollbar and not just for the track?

File: files/en-us/web/css/css_scrollbars/index.md [hunk 6]
```diff
@@ -1,58 +1,93 @@
 ---
-title: CSS Scrollbars
+title: CSS scrollbars styling
 slug: Web/CSS/CSS_Scrollbars
 page-type: css-module
+spec-urls: https://w3c.github.io/csswg-drafts/css-scrollbars/
 browser-compat:
-  - css.properties.scrollbar-width
   - css.properties.scrollbar-color
```
Comment (dipikabh) [inline files/en-us/web/css/css_scrollbars/index.md:7]: Can we put both back in? Adding BCD entry for one can raise questions about the support for the other property.

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks for the updates. Added two follow-up changes

File: files/en-us/web/css/css_scrollbars/index.md (hunk 6 above)
Comment (estelle) [reply files/en-us/web/css/css_scrollbars/index.md:7]: i would prefer to have neither. If we put them both in, it is huge. I'll remove them again ;)
Screen Shot 2023-04-26 at 11 33 12 AM

File: files/en-us/web/css/css_scrollbars/index.md [hunk 7]
```diff
@@ -1,67 +1,101 @@
...
+  - css.properties.scrollbar-width
 ---
 
-{{CSSRef}}
+{{CSSRef}}{{SeeCompatTable}}
 
-**CSS Scrollbars** standardizes the obsolete scrollbar color properties introduced in 2000 by Windows IE 5.5.
+The **CSS scrollbars styling** module defines properties that you can use for visual styling of scrollbars. You can  customize the width of the scrollbar as required. You can also customize the color of the scrollbar _track_, which is the background of the scrollbar, and the color of the scrollbar _thumb_, which is the draggable handle of the scrollbar.
```
Comment (dipikabh) [inline files/en-us/web/css/css_scrollbars/index.md:13]:
```suggestion
The **CSS scrollbars styling** module defines properties that you can use for visual styling of scrollbars. You can customize the width of the scrollbar as required. You can also customize the color of the scrollbar _track_, which is the background of the scrollbar, and the color of the scrollbar _thumb_, which is the draggable handle of the scrollbar.
```

**Review by dipikabh — DISMISSED** [review]

**Review by dipikabh — APPROVED** [review]

**Pull request merged**